datasets/imdb/README.md

```diff
@@ -3,6 +3,30 @@ pretty_name: IMDB
 languages:
 - en
 paperswithcode_id: imdb-movie-reviews
+train-eval-index:
+- config: plain_text
+  task: text-classification
+  task_id: binary_classification
+  splits:
+    train_split: train
+    eval_split: test
+  col_mapping:
+    text: text
+    label: label
+  metrics:
+  - type: accuracy
```

Comment:
I suggest adding a "pretty name" for all metrics so we can ultimately render this in AutoTrain
```suggestion
  - type: accuracy
  - name: Accuracy
```